src/operator/tensor/la_op-inl.h

```diff
@@ -921,6 +927,9 @@ struct det_backward {
     using namespace mshadow;
     using namespace mshadow::expr;
     using namespace mxnet_op;
+    if (dA.shape_.Size() == 0U) {
+      return;
+    }
     // compute inverse(A) and stores it to LU
     linalg_batch_det_backward_helper(LU, pivot, det, dA, DType(0), ctx);
     const_cast<Tensor<xpu, 3, DType>&>(dA) = broadcast_to(reshape(det * ddet, \
```

Comment:
@reminisce I believe the perpetrator to the flakiness on Windows is here. I was able to pinpoint the bug thanks to @haojin2 . I think this may corrupt memory sometimes.

Reply:
@arcadiaphy It was found that this line would cause flaky memory corruptions on Windows platforms (both CPU and GPU) only. Do you have any idea on where might be wrong? Thanks.

Reply:
@arcadiaphy It was found that this line would cause flaky memory corruptions on Windows platforms (both CPU and GPU) only. Do you have any idea on where might be wrong? Thanks.

Reply:
Fixed